### PR TITLE
magento/devdocs #8928 : Add description about data  import from configuration files

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/devdocs/issues/8928

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-deployment.html

## Links to Magento source code

-  https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-set.html
- https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html

### CODE REVIEW

File: src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-deployment.md [hunk 1]
```diff
@@ -52,6 +52,23 @@ The following table discusses the meanings of installation parameters and values
...
+
+After importing websites, stores, store views, and themes, you can create product attributes and apply them to websites, stores, and store views, on the production system.
+
+On your production system, run the following command to import data from the configuration files (`config.php` and `env.php`) to the database:
+
+```bash
+bin/magento app:config:import [-n, --no-interaction]
+```
+
+Use the optional `[-n, --no-interaction]` flag to import data without any interaction.
```
Comment (rogyar) [inline src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-deployment.md:68]: I would suggest simplifying this statement as follows 

```suggestion
Use the optional `[-n, --no-interaction]` flag to import data without additional confirmations.
```

... and removing the next "If you enter..." sentence

Comment (rogyar) [reply src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-deployment.md:68]: Also, we may put something like 

> For additional information, please, check the [Import data from configuration files](https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-config-mgmt-import.html)

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by rogyar** [comment]
Hi @EvheniiaKolkhiieva. Thank you for your contribution. Please, check my minor suggestions above

**Comment by EvheniiaKolkhiieva** [comment]
Hi @rogyar, thank you for reviewing.
I have implemented your suggestions, could you please approve?

**Review by rogyar — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**